flags.go

```diff
@@ -177,6 +177,13 @@ func newApp() (app *cli.App) {
 					"(use -1 for no limit)",
 			},
 
+			cli.IntFlag{
+				Name:  "sequential-read-size-mb",
```

Comment:
just curious, why sequential and not single? This bounds the maximum single read call bytes that we request from GCS?

Reply:
For random, maxReadSize is 8MB https://github.com/GoogleCloudPlatform/gcsfuse/blob/fa9807bd3d424e8f398c3eb059038fe772052bc4/internal/gcsx/random_reader.go#L56.

since it is used only for sequential flows, named it as sequential